exchanges/bybit/bybit_wrapper.go

```diff
@@ -2147,3 +2155,206 @@ func (by *Bybit) UpdateOrderExecutionLimits(ctx context.Context, a asset.Item) e
 	}
 	return by.LoadLimits(limits)
 }
+
+// GetFuturesContractDetails returns details about futures contracts
+func (by *Bybit) GetFuturesContractDetails(ctx context.Context, item asset.Item) ([]futures.Contract, error) {
+	if !item.IsFutures() {
+		return nil, futures.ErrNotFuturesAsset
+	}
+	if !by.SupportsAsset(item) {
+		return nil, fmt.Errorf("%w %v", asset.ErrNotSupported, item)
+	}
+
+	inverseContracts, err := by.GetInstrumentInfo(ctx, "inverse", "", "", "", "", 1000)
+	if err != nil {
+		return nil, err
+	}
+	linearContracts, err := by.GetInstrumentInfo(ctx, "linear", "", "", "", "", 1000)
```

Comment:
This is not called in `CoinMarginedFutures` so this can be shifted to the switch case as needed